students/etwum/lesson03/mailroom.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+if __name__ == "__main__":
+    print()
```

Comment:
for readability and consistency (if __name__ usually goes at the bottom) I would move this down at the bottom of the script and have the main guts there, calling support functions and no extra code outside of that.